### DISCUSSION TITLE
buddhism/2024/emptiness/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#126] [General]
# buddhism/2024/emptiness/

https://shanenull.com/buddhism/2024/emptiness/

**Comment by shane0** [comment]
# shunyata

>  everything is interconnected and changing together
>> the path is to do this also but with more skillful means and in harmony with everything with less suffering and more love and without attachment

- emptiness (sunyata): The fundamental teaching of Mahayana Buddhism, signifying the lack of suabhava, or self-nature, inherent nature, independent being, self-defining essence, or invariant identity of anything.
- All phenomena (dharmas) occur only in dependence on causes and conditions. 
- See also dependent arising; Mahayana; self-nature.

**Reply by shane0** [reply]
Everything is interconnected and interdependent, and nothing exists on its own.

**Reply by shane0** [reply]
> Anything changing is not real

**Comment by shane0** [comment]
# emptiness defined

- https://84000.co/translation/toh21

emptiness
stong pa nyid
སྟོང་པ་ཉིད།
śūnyatā

Definition from the 84000 Glossary of Terms:

Emptiness denotes the ultimate nature of reality, the total absence of inherent existence and self-idetity with respect to all phenomena. 

According to this view, all things and events are devoid of any independent, intrinsic reality that constitutes their essence. 

Nothing can be said to exist independent of the complex network of factors that gives rise to its origination, nor are phenomena independent of the cognitive processes and mental constructs that make up the conventional framework within which their identity and existence are posited. 

When all levels of conceptualization dissolve and when all forms of dichotomizing tendencies are quelled through deliberate meditative deconstruction of conceptual elaborations, the ultimate nature of reality will finally become manifest. 

It is the first of the three gateways to liberation.

Located in 8 passages in the translation:

s.­1 1.­6-8 n.­36 n.­43-44 n.­49

**Reply by shane0** [reply]
https://youtu.be/L069esCjCR0

**Comment by shane0** [comment]
# change and time

There’s nothing to change energy is not created or destroyed

Energy energy dissipates

Entropy dissipates and drives time

Everything is changing and nothing is changing

https://youtu.be/ooo_tCZ5YNk?si=F4BS5y6SFOjBGGp8

**Reply by shane0** [reply]
https://shanenull.com/stoicism/entropy/

**Comment by shane0** [comment]
# edward conze

- https://shanenull.com/buddhism/2025/edward/
- I am reading his books & he has more insight & better translations
- shunyata sanskrit root words mean swollen / empty / hollow
- synonyms are non-dual

> From another point of view emptiness is called Suchness, because one takes reality such as it is, without superimposing any ideas upon it.

**Reply by shane0** [reply]
# swollen

- sanskrit for hollow
- no central self inside
- swollen with the five skhandas
- or filled with something foreign
- feynman: everything is atoms
- suzuki: everything is changing
